packages/connect-conformance/src/web/client.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright 2021-2024 The Connect Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { ClientCompatResponse } from "../gen/connectrpc/conformance/v1/client_compat_pb.js";
+import { Buffer } from "node:buffer";
+import { remote } from "webdriverio";
+import type { RemoteOptions } from "webdriverio";
+import * as esbuild from "esbuild";
+import { parseArgs } from "node:util";
+import { readSizeDelimitedBuffers } from "../protocol.js";
+
+const { values: flags } = parseArgs({
+  args: process.argv.slice(2),
+  options: {
+    browser: { type: "string", default: "chrome" },
+  },
+});
+
+export async function run() {
+  let capabilities: RemoteOptions["capabilities"] = {
+    acceptInsecureCerts: true,
+  };
+  switch (flags.browser) {
+    case "chrome":
+    case undefined:
+      capabilities = {
+        ...capabilities,
+        browserName: "chrome",
+        "goog:chromeOptions": {
+          args: ["--disable-gpu", "--headless"],
```

Comment:
Similar to the log output (comment below), it can be necessary to run tests in a non-headless browser with access to the network inspector and browser console. 

For context, here is documentation for running in a full local browser with the old crosstests: https://github.com/connectrpc/connect-es/tree/main/packages/connect-web-test#running-tests-in-a-local-browser

How can we do the same with webdriver?